providers/snowflake/schema.go

```diff
@@ -24,9 +26,14 @@ type SchemaGenerator struct {
 func (g SchemaGenerator) createResources(schemaList []schema) []terraform_utils.Resource {
 	var resources []terraform_utils.Resource
 	for _, schema := range schemaList {
+		name := schema.Name.String
+		if name == "INFORMATION_SCHEMA" || name == "PUBLIC" {
```

Comment:
maybe leave a comment why
"These schemas are always managed by snowflake, skp them" or something like that